### DISCUSSION TITLE
Add ability to hide "create new" and "attach"/"detach" buttons when viewing associations.

### DISCUSSION THREAD

**Discussion by calebuharrison** [avo-hq/avo#400] [Feature requests]
Sometimes, I only want resources to be viewed through their associations to other resources, which can make the "attach" and "detach" buttons a bit confusing in that context. It would be great if there was a mechanism for hiding those.

**Comment by adrianthedev** [comment]
This is definitely something we can release next week.

**Comment by adrianthedev** [comment]
This has been released in `1.9` 💪
